app/controllers/placements/support/provider_suggestions_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Placements::Support::ProviderSuggestionsController < Placements::Support::ApplicationController
+  def index
+    accredited_providers = AccreditedProviderApi.call
+    filtered_providers = filter_providers(accredited_providers, params[:query])
```

Comment:
Is it worth doing `params.require(:query)`?

Reply:
resolved in [abdfae3](https://github.com/DFE-Digital/itt-mentor-services/pull/48/commits/abdfae3e4ab05b2c4cf0a45a0c634fec1fbf6545)